### PR TITLE
Gists endpoints get

### PR DESCRIPTION
**Pull request by Henning-K**
crossed off endpoints.
Partial fix for https://github.com/mgattozzi/github-rs/issues/89

### CODE REVIEW

File: src/gists/get.rs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    @GistsGistId
+        |=> comments -> GistsGistIdComments
+        |
+    @GistsGistIdComments
+        |
+        |=> Id -> GistsGistIdCommentsId = Id_str
+);
+
+exec!(Gists);
+
```
Comment (mgattozzi) [inline src/gists/get.rs]: Can you get rid of this extra space here?

File: src/gists/get.rs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+exec!(GistsGistIdComments);
+exec!(GistsGistIdCommentsId);
+
```
Comment (mgattozzi) [inline src/gists/get.rs]: Same with this one

File: src/gists/get.rs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+exec!(GistsId);
+exec!(GistsIdCommits);
+exec!(GistsIdForks);
+exec!(GistsIdSha);
+exec!(GistsIdStar);
+
```
Comment (mgattozzi) [inline src/gists/get.rs]: And lastly this one!

**Comment by mgattozzi** [comment]
@Henning-K same as the delete PR just squash these and change the PR title to "Add GET endpoints for Gists"

**Comment by mgattozzi** [comment]
@Henning-K agh I can't believe I forgot to tell you this so it's my bad. You'll need to put a `func_client!` macro for each of these like in this [line](https://github.com/mgattozzi/github-rs/blob/master/src/client.rs#L232). It'll be one for each type of endpoint (GET, PUT, POST etc.) so for this PR under GetQueryBuilder in client.rs you put `func_client!(gists, gists::get::Gists<'g>);` Make sense? You'll need to do this for each PR and squash it again with these changes. Sorry about that :(

**Comment by Henning-K** [comment]
Done for all five PRs.

**Comment by mgattozzi** [comment]
@Henning-K last nit pick and I'll merge :D Can you get rid of the extra spaces between each of the `exec!()` macros?

**Comment by Henning-K** [comment]
Should I rebase/squash this one too?

**Comment by mgattozzi** [comment]
Alright just waiting for Travis to finish up here @Henning-K and after that I'll merge this. Thanks for all the hard work and knocking out that entire set of endpoints really quickly. I greatly appreciate it! :D

**Pull request merged**